src/legacy/core_plugins/kbn_vislib_vis_types/public/controls/point_series/value_axes.js

```diff
@@ -1,205 +0,0 @@
-/*
- * Licensed to Elasticsearch B.V. under one or more contributor
- * license agreements. See the NOTICE file distributed with
- * this work for additional information regarding copyright
- * ownership. Elasticsearch B.V. licenses this file to you under
- * the Apache License, Version 2.0 (the "License"); you may
- * not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *    http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing,
- * software distributed under the License is distributed on an
- * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
- * KIND, either express or implied.  See the License for the
- * specific language governing permissions and limitations
- * under the License.
- */
-
-import _ from 'lodash';
-import { uiModules } from 'ui/modules';
-import vislibValueAxesTemplate from './value_axes.html';
-import { safeMakeLabel } from 'ui/agg_types/agg_utils';
-
-const module = uiModules.get('kibana');
-
-module.directive('vislibValueAxes', function () {
-  return {
-    restrict: 'E',
-    template: vislibValueAxesTemplate,
-    replace: true,
-    link: function ($scope) {
-      let isCategoryAxisHorizontal = true;
-
-      function mapPosition(position) {
-        switch (position) {
-          case 'bottom': return 'left';
-          case 'top': return 'right';
-          case 'left': return 'bottom';
-          case 'right': return 'top';
-        }
-      }
-
-      function mapPositionOpposite(position) {
-        switch (position) {
-          case 'bottom': return 'top';
-          case 'top': return 'bottom';
-          case 'left': return 'right';
-          case 'right': return 'left';
-        }
-      }
-
-      $scope.rotateOptions = [
-        { name: 'horizontal', value: 0 },
-        { name: 'vertical', value: 90 },
-        { name: 'angled', value: 75 },
-      ];
-
-      $scope.$watch('editorState.params.categoryAxes[0].position', position => {
-        isCategoryAxisHorizontal = ['top', 'bottom'].includes(position);
```

Comment:
**note:** moved to https://github.com/elastic/kibana/pull/43772/files?file-filters%5B%5D=.tsx&hide-deleted-files=true#diff-af178345ae0266d7293defbfeef8b57bR227